bigbluebutton/greenlight
# BSN-068: Increased local account password security

## Description


## Testing Steps

There are a couple cases that need to be tested:
1. Log in with an existing local greenlight account -> you should be redirected to the reset_password page 
2. Reset your password and make sure you can log in again
3. While logged in, mark your password as insecure again `User.find_by(email: ...).update(secure_password: false)`
4. Change your password via the profile password update and make sure that you are not redirected to the reset_password page next time you login
5. Create a new account, then log out. Make sure you are not redirected to the reset_password page when you login


## Screenshots (if appropriate):
